src/pages/index.js

```diff
@@ -57,12 +113,32 @@ editProfileFormValidator.enableValidation();
 const addCardFormValidator = new FormValidator(validationSettings, addCardForm);
 addCardFormValidator.enableValidation();
 
+const avaFormValidator = new FormValidator(validationSettings, avaEditForm);
+avaFormValidator.enableValidation();
+
 //user info
 const userInfo = new UserInfo({
   userNameSelector: ".profile__title",
-  userTitleSelector: ".profile__description",
+  userDescriptionSelector: ".profile__description",
+  userAva: ".profile__image",
 });
+const avatarModal = new ModalWithForm(selectors.changeAvaModal, handleAvaModal);
 
+function handleAvaModal(inputValues) {
+  avatarModal.renderLoading(true);
+  api
+    .updateProfileAvatar(inputValues.avatar)
+    .then(() => {
+      userInfo.setAvatar(inputValues.avatar);
+      avatarModal.close();
+    })
+    .catch((err) => {
+      console.log(err);
+    })
+    .finally(() => {
```

Comment:
The api requests are implemented correctly. Everything are on the right places

Reply:
Great!